app/components/dialog/components/modifyProfile/index.tsx

```diff
@@ -55,27 +57,24 @@ const validateForm = (values: ModifyProfileFormState): FormErrors<ModifyProfileF
 };
 
 @withStyles<typeof ModifyProfileDialog>(styles)
-class ModifyProfileDialog extends React.PureComponent<
-  ModifyProfileProps & InjectedFormProps<ModifyProfileFormState, ModifyProfileProps>
-> {
-  public constructor(props: ModifyProfileProps & InjectedFormProps<ModifyProfileFormState, ModifyProfileProps>) {
+class ModifyProfileDialog extends React.PureComponent<ModifyProfileProps> {
+  private formikNode: Formik<ModifyProfileFormState> | null;
+  public constructor(props: ModifyProfileProps) {
```

Comment:
You don't need constructor because this component doesn't have React's component state anymore.